hamocc/mo_kind.F90

```diff
@@ -0,0 +1,46 @@
+! Copyright (C) 2025  J. Maerz, J. Schwinger, T. Torsvik
+!
+! This file is part of BLOM/iHAMOCC.
+!
+! BLOM is free software: you can redistribute it and/or modify it under the
+! terms of the GNU Lesser General Public License as published by the Free
+! Software Foundation, either version 3 of the License, or (at your option)
+! any later version.
+!
+! BLOM is distributed in the hope that it will be useful, but WITHOUT ANY
+! WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+! FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for
+! more details.
+!
+! You should have received a copy of the GNU Lesser General Public License
+! along with BLOM. If not, see https://www.gnu.org/licenses/.
+
+module mo_kind
+
+  use mod_types,   only: blom_i1 => i1, blom_i2 => i2, blom_i4 => i4, blom_i8 => i8, &
+                         blom_r4 => r4, blom_r8 => r8, blom_kind_cs, blom_kind_cm,   &
```

Comment:
The renaming `=>` of BLOM names here does not make sense, if this is done explicitly below. I would stick with the explicit way below and remove this here.

Reply:
I don't think we can do 
```
use mod_types, only: i1, i2, ...

integer, parameter :: i1 = i1
```
Alternatively, we could import `int8`, `int16` ... from `iso_fortran_env`, as is done in `mod_types`.

Reply:
Ah ok, I misunderstood this. Sorry, this is fine as it was